app/views/comments/_form.html.erb

```diff
@@ -40,24 +40,23 @@
   </div>
   <div class="code-of-conduct" id="toggle-code-of-conduct-checkbox">
   </div>
-  <a href="/p/editor_guide" class="markdown-guide" target="_blank" title="Markdown Guide">
-    <img alt="markdown guide" class="icon-image" src="<%= asset_path("info.svg") %>" />
-  </a>
-  <div class="editor-image-upload">
-    <input type="file" id="image-upload-main" name="file" accept="image/*" style="display:none">
-    <button class="image-upload-button" id="image-upload-button-main" onclick="handleImageUpload(event,'main')" title="Upload Image">
-      <img alt="upload image" class="icon-image" src="<%= asset_path("image-upload.svg") %>" />
-    </button>
-    <label class="image-upload-file-label" id="image-upload-file-label-main"></label>
-    <input type="submit" id='image-upload-submit-main' value="Upload" style="display:none">
-    <input class="uploaded-image" id="uploaded-image-main" />
-  </div>
-  <div class="actions" id="submit-wrapper">
-    <%= link_to(request.referer.present? ? :back : commentable.path, "aria-label": "Cancel action") do %>
-      <button id="cancel-button" class="comment-action-button comment-action-cancel">CANCEL</button>
-    <% end %>
-    <button id="preview-button" class="comment-action-button comment-action-preview">PREVIEW</button>
-    <%= f.submit "SUBMIT", onclick: "validateField(event)", class: "comment-action-button", id: "submit-button" %>
+  <div class="edit-comment-footer">
```

Comment:
The only real change hear is wrapping the footer stuff of the comment form in a `div`, i.e. `<div class="edit-comment-footer" />`. Not sure if semantically this should be `<footer />` since there is no `<header />`.